src/qiskit_iqm/fake_backends/iqm_fake_backend.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2022-2023 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=no-name-in-module,import-error,super-init-not-called,too-many-instance-attributes
+"""
+Fake backend for simulating IQM quantum computers.
+"""
+
+from typing import Optional, Union
+
+from qiskit import QuantumCircuit
+from qiskit.providers import JobV1, Options
+from qiskit_aer import AerSimulator
+from qiskit_aer.noise import NoiseModel
+from qiskit_aer.noise.errors import depolarizing_error, thermal_relaxation_error
+
+from qiskit_iqm.fake_backends.chip_sample import IQMChipSample
+from qiskit_iqm.iqm_backend import IQM_TO_QISKIT_GATE_NAME, IQMBackendBase
+
+
+class IQMFakeBackend(IQMBackendBase):
+    """
+    Fake backend for simulating an IQM QPU.
```

Comment:
The docstring should explain what a fake backend does and how it differs from a real backend.

Reply:
Addressed in [e322592](https://github.com/iqm-finland/qiskit-on-iqm/pull/35/commits/e32259246bf934e4ce1e33d51777434d2f3423a5).